client.lua

```diff
@@ -49,8 +54,21 @@ if Config.UseTarget then
                 end
             }
         }
-    })
-else
+        exports.ox_target:addLocalEntity(electricalBoxEntity, options)
+    end
+end
+
+local function removeElectricalBox()
+    if Config.UseTarget then
+        exports.ox_target:removeLocalEntity({electricalBoxEntity}, 'qb-jewelery:electricalBox')
```

Comment:
```suggestion
        exports.ox_target:removeLocalEntity(electricalBoxEntity, 'qb-jewelery:electricalBox')
```
A single entity is removed here

Reply:
Amended syntax for removeLocalEntity entity.